montecarlo/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
-add_subdirectory(eg)
-add_subdirectory(vmc)
-if(pythia6)
+if(geo)
+  add_subdirectory(eg)
+  add_subdirectory(vmc)
+  if(pythia6)
   add_subdirectory(pythia6)
```

Comment:
Do the really need geom?

Reply:
Yes, through VMC.

Reply:
it also makes sense to have:
if (vmc) add_subdirectory(geo)
Can one describe dependencies in cmake and use those?

Reply:
do you mean to have a nested system of vmc->geo->other fine grained options?